packages/slice-machine/pages/index.tsx

```diff
@@ -158,28 +158,35 @@ const CustomTypes: React.FunctionComponent = () => {
         breadrumbHref="/"
       />
       {customTypes.length === 0 ? (
-        <EmptyState
-          title={"Create your first Custom Type"}
-          explanations={[
-            "Click the + button on the top right to create your first custom type.",
-            "It will be stored locally. You will then be able to push it to Prismic.",
-          ]}
-          onCreateNew={openCreateCustomTypeModal}
-          buttonText={"Create your first Custom Type"}
-          documentationComponent={
-            <>
-              Go to our{" "}
-              <ThemeLink
-                target={"_blank"}
-                href={"https://prismic.io/docs/core-concepts/custom-types "}
-                sx={(theme) => ({ color: theme?.colors?.primary })}
-              >
-                documentation
-              </ThemeLink>{" "}
-              to learn more about Custom Types.
-            </>
-          }
-        />
+        <Flex
+          sx={{
+            flex: 1,
+            justifyContent: "center",
+            alignItems: "center",
+          }}
+        >
+          <EmptyState
```

Comment:
I don't know as the flex around is here for positioning the empty state inside the page itself, if we do so that mean this component will always be in full page in the middle of his parent. It's a choice to do between having a more reusable component or a more context tied component. 

What is your preference ?

Reply:
We keep like that :)